Scripts/install.sh

```diff
@@ -4,13 +4,14 @@ set -euo pipefail
 
 DOWNLOAD_URI=https://github.com/alcatraz/Alcatraz/releases/download/1.1.15/Alcatraz.tar.gz
 PLUGINS_DIR="${HOME}/Library/Application Support/Developer/Shared/Xcode/Plug-ins"
+XCODE_APP_PATH="$(xcode-select -p | sed s/\.app.*/.app/)"
```

Comment:
We can bypass this altogether by using `defaults read com.apple.dt.Xcode "$PLIST_PLUGINS_KEY"` instead of `-app`. Both beta and standard Xcode use this bundle identifier.